src/common/network/NetworkUtils.cpp

```diff
@@ -230,6 +230,9 @@ StatusOr<HostAddr> NetworkUtils::toHostAddr(folly::StringPiece ipPort) {
 }
 
 StatusOr<std::vector<HostAddr>> NetworkUtils::toHosts(const std::string& peersStr) {
+    if (peersStr.empty()) {
```

Comment:
I think we should NOT forbid empty peersStr.  
Because in some cases,  we actually pass in empty string.